app/models/image.rb

```diff
@@ -0,0 +1,3 @@
+class Image < ApplicationRecord
+  validates :link, format: URI.regexp(%w[http https])
```

Comment:
There are several ways to "validate" a link, and there are many gems that do this in different ways. To try your hand at adding gems to your project, try validating the url using https://github.com/perfectline/validates_url (as a separate commit).